.buildkite/pipeline.yml

```diff
@@ -1,6 +1,42 @@
+# For convenience with the many install_swiftpm_dependencies steps, we declare agents and env for macOS in the root, then specialize the fewer remaining steps
+agents:
+  queue: mac
+env:
+  IMAGE_ID: xcode-15.4
+
 steps:
   - label: "🕵️ Lint"
     command: make lint
+    agents:
+      queue: default
 
   - label: "🔬 Test"
     command: make test
+    agents:
+      queue: default
```

Comment:
By the way, these have been running on the `upload` queue all this time because of how the pipeline was configured in the IaC setup.

Reply:
Ooof. Nice catch.